docs/icons.md

```diff
@@ -4,10 +4,28 @@ This document will cover how to use icons in Calypso, as well as how to create i
 
 ## Gridicons
 
-Gridicons is the brand new icon set designed from the bottom up for Calypso. It features a consistent style.
+[Gridicons](https://github.com/automattic/gridicons) is the brand new icon set designed from the bottom up for Calypso. It features a consistent style. There's a gallery with all the available icons in https://automattic.github.io/gridicons/.
```

Comment:
I would just remove the sentence about it being the "brand new icon set" and the one about consistency being a "feature" :)

Reply:
@drw158 Thanks, I agree. I just pushed up a copy change to that section.